AnastasiyaTomasheva/HTML_CSS/index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Organik</title>
+    <link rel="stylesheet" href="./styles/media.css">
```

Comment:
your media dont working because they connect before style.css. Style.css overlap your media.css. Please check your code in browser using inspect(tab Styles into Elements)